.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+<!--
+Here are some checklists you may like to use. Use your judgement.
+
+If you're new to the codebase, or you have questions, CONTRIBUTING has many answers.
+-->
+Pre-submit checklist:
+- [ ] Key commits have useful messages
+- [ ] Relevant tickets are mentioned in commit messages
+- [ ] Useful pull request description
```

Comment:
Useful to who? I'd be more specific. E.g. "Provides context for reviewing".

Reply:
Same response as the previous point.